aml-development/frontend
# fix(listing): remove actions from deleted listing tile

AMLOS-829

**Description**
From Listing Manangement Tile View, look at Delete Listings.

Listings have all of the same functionality available as a normal listing. (View, Launch, Edit, Delete).

**Acceptance Criteria**
Functionality is removed from Deleted Listing

**How to Test**
Pull this branch
Delete a listing (you can do this from the Listing Management section when logged in as bigbrother)
Verify that the deleted listing tile does not have any action buttons and cannot be clicked to view the listing
